ecs/spotfleet-cluster/README.md

```diff
@@ -0,0 +1,17 @@
+# ECS Spotfleet Cluster
```

Comment:
In addition to this, I think you should add it to [this](https://github.com/TeliaSoneraNorge/telia-terraform-modules/blob/spotfleet/ecs/README.md) readme, and perhaps make a note of "make sure to read the extended readme" for the module.